5.0.0/buster-slim/Dockerfile

```diff
@@ -34,4 +34,8 @@ WORKDIR $IDSVR_HOME
 COPY --chown=idsvr idsvr-5.0.0/idsvr /opt/idsvr
 COPY --chown=idsvr first-run /opt/idsvr/etc/first-run
 
+RUN find /opt/idsvr -type f -exec chmod a-w {} \; && \
+	chmod -R o-rwx /opt/idsvr && \
+	chown -R idsvr: /opt/idsvr
```

Comment:
`chown -R idsvr: /opt/idsvr` can be removed if in line 34 and 35 we do `COPY --chown=idsvr:idsvr ...` instead

Reply:
Applied changed here.